docs/project/2024/alberto.udrea0412/index.md

```diff
@@ -0,0 +1,78 @@
+# Weather Station
+
+:::info 
+
+**Author**: Udrea Alberto-George \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-UdrAlberto
+
+:::
+
+## Description
+
+The project builds a weather station which should provide real-time weather data on an external display and web interface
+
+## Motivation
+
+Building a Raspberry Pi Pico W weather station with embassy-rs combined my interest in embedded systems with a chance to explore networking. It allows me to learn sensor interaction, data processing, and real-time programming. Integrating a web interface adds a network layer, enabling real-time data display and opening doors for future functionalities like remote monitoring.
+
+## Architecture 
+
+![Architecture](architecture_light.svg#gh-light-mode-only)]![Architecture](architecture_dark.svg#gh-dark-mode-only)
+
```

Comment:
I tried adding theme-specific files here, but their visibility doesn't seems to be affected by it resulting in both images being visible at once. I used the GitHub image theming approach, should I have used something else?